java/client/src/main/java/glide/api/models/GlideString.java

```diff
@@ -95,4 +95,12 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Arrays.hashCode(bytes);
     }
+
+    // Method to concatenate two GlideString objects
+    public GlideString concat(GlideString other) {
+        byte[] concatenatedBytes = new byte[this.bytes.length + other.bytes.length];
+        System.arraycopy(this.bytes, 0, concatenatedBytes, 0, this.bytes.length);
+        System.arraycopy(other.bytes, 0, concatenatedBytes, this.bytes.length, other.bytes.length);
+        return GlideString.of(concatenatedBytes);
```

Comment:
I think you can simplify this
```suggestion
        return of(concatenateArrays(this.bytes, other.bytes));
```

Reply:
This does does work because concatenateArrays does not work with primitive types like `bytes[]`